2-environments/envs/development/outputs.tf

```diff
@@ -38,3 +38,8 @@ output "env_secrets_project_id" {
   description = "Project for environment related secrets."
   value       = module.env.env_secrets_project_id
 }
+
+output "monitoring_workspace_users" {
```

Comment:
Rather than plumbing this for the test, can we retrieve this in test from `TF_VAR_group_email` using `utils.ValFromEnv`?